### PR TITLE
c28 translation

### PR DESCRIPTION
**Pull request by ManlyMarco**
Thanks to @ScrewThisNoise for providing the translations.

### CODE REVIEW

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c28/00/1008/translation.txt [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-//「……美しい」=
-//「海ほたる達の輝きに向ける言葉など、きっと……それくらいで充分なのじゃ」=
-//「あはは。随分と大人びたこと言うよね？」=
-//「わしはいつも大人なのじゃぞ？　まぁ、この場の雰囲気には合ってなかったかもしれぬが……」=
-//「なら、ここからは初々しい恋人同士ってことで」=
-//「……うむ、海ほたる達が彩ってくれたこの夜にしばし浸るとするのじゃ」=
-//「キレイだね」=
-//「…………そうじゃな、本当に」=
+「見よ、懸命に瞬いておる。このほの暗い世界で己の存在を主張しようとするかのようじゃ」=“Look, they are twinkling so brightly, making their existence known to this dark and gloomy world.”
+「俺はここにいるぞー、って？」=“I'm (ore) right here, you know!?”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c28/00/1008/translation.txt:14]: Is `(ore)` here just a translation note?

**Review by GeBo1 — COMMENTED** [review]
Question on this one

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c28/00/1008/translation.txt (hunk 1 above)
Comment (ManlyMarco) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c28/00/1008/translation.txt:14]: Yeah, looks like a leftover note.

Comment (ManlyMarco) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c28/00/1008/translation.txt:14]:
```suggestion
「俺はここにいるぞー、って？」=“I'm right here, you know!?”
```

**Review by GeBo1 — APPROVED** [review]

**Pull request merged**